### PR TITLE
ekf2: height aid source consistency

### PR DESCRIPTION
**Pull request by dagar**
- the goal here is to keep all height source (baro/EV/GNSS/RNG) handling as consistent as possible, possibly capturing most of this in a class later

### CODE REVIEW

**Comment by dagar** [comment]
Tested (as part of https://github.com/PX4/PX4-Autopilot/pull/19128) https://logs.px4.io/plot_app?log=509328a3-8719-4297-98cf-8b0b5a1d7d64

**Review by bresch — APPROVED** [review]
Hopefully we can soon extract them into classes and refactor this because I have the impression to loose the high-level picture of what's happening when all the implementation details are in the main function like that...

File: src/modules/ekf2/EKF/range_height_control.cpp [hunk 1]
```diff
@@ -32,168 +32,174 @@
  ****************************************************************************/
 
 /**
- * @file gps_control.cpp
+ * @file range_height_control.cpp
```
[the remark this answers is not in the document]
Comment (dagar) [reply src/modules/ekf2/EKF/range_height_control.cpp:35]: I personally don't think so.

**Comment by dagar** [comment]
Removed auto change_indication/iris_gps.csv update to resolve merge conflict.

**Pull request merged**